cscs-checks/cuda/multi_gpu.py

```diff
@@ -41,15 +43,22 @@ def __init__(self):
         if self.current_system.name in ['daint', 'dom', 'tiger']:
             self.modules = ['craype-accel-nvidia60']
             self.num_gpus_per_node = 1
-        else:
-            self.modules = ['craype-accel-nvidia35']
+        elif self.current_system.name == 'kesch':
+            self.modules = ['cudatoolkit/8.0.61']
+            self.num_gpus_per_node = 8
+        elif self.current_system.name in ['arolla', 'tsa']:
+            self.modules = ['cuda/10.1.243']
             self.num_gpus_per_node = 8
 
         # perf_patterns and reference will be set by the sanity check function
         self.sanity_patterns = self.do_sanity_check()
         self.perf_patterns = {}
         self.reference = {}
         self.__bwref = {
+# FIXME: reference values for Arolla and Tsa need to be updated (sanity check fails if they are not defined)
```

Comment:
Can you align this at the indentation level inside the dictionary? And also wrap the message at the 80 columns?